client/src/components/Dataset/DatasetHistory.vue

```diff
@@ -1,21 +1,26 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { type components } from "@/api/schema";
+import { useHistoryStore } from "@/stores/historyStore";
+
+type HDASummary = components["schemas"]["HDASummary"];
```

Comment:
In general, it's better to export this type from "@/api/datasets" and reuse it in other components instead of redefining it each time.